packages/okta-angular/README.md

```diff
@@ -94,6 +94,10 @@ For PKCE flow, this should be left undefined or set to `['code']`.
 - `autoRenew` *(optional)*:
   By default, the library will attempt to renew expired tokens. When an expired token is requested by the library, a renewal request is executed to update the token. If you wish to  to disable auto renewal of tokens, set autoRenew to false.
 
+- `onTokenError` *(optional)* - callback function. If there is an error while renewing a token, the error will be passed to a handler function. The default handler calls `loginRedirect()` to initiate a login flow. Passing a function here will override the default handler.
```

Comment:
Is this an error in renewing (`onRenewError()`?) or an error with the token in general?  If the latter, the documentation does not match that.

Reply:
It is an error emitted from the TokenManager. Currently, Renew is the only case where this error will be emitted. Would `onTokenManagerError` be better?

Reply:
If it's for all errors and the renew is all that we have now, that's fine as is, I just couldn't tell if we expected future errors from the token manager to come through this callback from the text, so I wanted to be sure.

Reply:
We could instead have a callback ONLY for the "login_required" error code. (One could still `getTokenManager().on('error', ...` to listen for other errors should they ever exist) This would be almost identical in purpose to the existing callback `onAuthRequired`. In fact we could use that same callback.